From 31617a199faba7f2de9620777ede3b10c35618f5 Mon Sep 17 00:00:00 2001
From: Frank van der Linden <fllinden@amazon.com>
Date: Fri, 13 Sep 2019 18:38:00 +0000
Subject: lustre: fix lnet makefile

Fix the lnet Makefile to use obj- instead of subdir-

Signed-off-by: Frank van der Linden <fllinden@amazon.com>
---
 drivers/staging/lustrefsx/lnet/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/staging/lustrefsx/lnet/Makefile b/drivers/staging/lustrefsx/lnet/Makefile
index 85a225e57b29..7ee52eb55902 100644
--- a/drivers/staging/lustrefsx/lnet/Makefile
+++ b/drivers/staging/lustrefsx/lnet/Makefile
@@ -1,3 +1,3 @@
-subdir-$(CONFIG_LUSTREFSX_LNET)			+=	lnet/
-subdir-$(CONFIG_LUSTREFSX_LNET)			+=	klnds/
-subdir-$(CONFIG_LUSTREFSX_LNET_SELFTEST)	+=	selftest/
+obj-$(CONFIG_LUSTREFSX_LNET)			+=	lnet/
+obj-$(CONFIG_LUSTREFSX_LNET)			+=	klnds/
+obj-$(CONFIG_LUSTREFSX_LNET_SELFTEST)	+=	selftest/
-- 
2.32.0

